testsuite/elytron/src/test/java/org/wildfly/test/integration/elytron/sasl/mgmt/DigestSha384MgmtSaslTestCase.java

```diff
@@ -0,0 +1,53 @@
+package org.wildfly.test.integration.elytron.sasl.mgmt;
```

Comment:
This needs a copyright header.

Reply:
@jamezp Copyright header was added to this test